R/etwfe.R

```diff
@@ -231,12 +249,23 @@ etwfe = function(
   
   ## Overload class and new attributes (for post-estimation) ----
   class(est) = c("etwfe", class(est))
-  attr(est, "etwfe") = list(
-    gvar = gvar,
-    tvar = tvar,
-    gref = gref,
-    tref = tref
-  )
+  if (!is.null(intvar)){
+    attr(est, "etwfe") = list(
+      gvar = gvar,
+      tvar = tvar,
+      gref = gref,
+      tref = tref,
+      intvar = intvar
+      
+    )
+  } else {
+    attr(est, "etwfe") = list(
+      gvar = gvar,
+      tvar = tvar,
+      gref = gref,
+      tref = tref
+    ) 
+  }
```

Comment:
I don't think we need the if...else condition here. If intvar (xvar) is NULL then it's still fine to include it in the return list since it won't affect the downstream methods.